archethic-foundation/archethic-node
# Complete burn of the transaction fee

# Description

This feature provides removal of `node_movements` in the transaction validation stamp and also complete burn of the transaction's fee, so the node rewards will not be replicated during the transaction validation. 

Fixes #163 #124 

## Type of change

- New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Check on the transaction explorer to not see any node movements
- Check on the explorer, the node doesn't get reward inputs after tx validation

# Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules
